addons/payment/controllers/post_processing.py

```diff
@@ -44,8 +45,8 @@ def poll_status(self, **_kwargs):
         monitored_tx = request.env['payment.transaction'].sudo().browse(
             self.get_monitored_transaction_id()
         ).exists()
-        if not monitored_tx:  # The session might have expired, or the tx has never existed.
-            raise Exception('tx_not_found')
+        if not monitored_tx:
+            raise ValidationError(_('The session might have expired, or transaction is not found'))
```

Comment:
raise ValidationError('tx_not_found')
also don't use translation because it handle in js side here https://github.com/odoo/odoo/blob/c4116f435fd704d21eb3e1c87f9f6184b9552689/addons/payment/static/src/js/post_processing.js#L60